packages/libs/client-utils/etc/client-utils-built-in.api.md

```diff
@@ -4,15 +4,45 @@
 
 ```ts
 
+import type { ChainId } from '@kadena/client';
+import { ICommand } from '@kadena/types';
+import { ICommandResult } from '@kadena/chainweb-node-client';
+import { ILocalCommandResult } from '@kadena/chainweb-node-client';
 import type { INetworkOptions } from '@kadena/client';
-import type { IPactCommand } from '@kadena/client';
+import { IPactCommand } from '@kadena/client';
+import { IPactDecimal } from '@kadena/types';
+import { IPactInt } from '@kadena/types';
 import type { ISignFunction } from '@kadena/client';
+import { ITransactionDescriptor } from '@kadena/client';
+import { PactValue } from '@kadena/types';
+
+// Warning: (ae-forgotten-export) The symbol "IDeployContractInput" needs to be exported by the entry point index.d.ts
+//
+// @alpha (undocumented)
+export const createPactCommand: (inputs: IDeployContractInput) => Partial<IPactCommand>;
```

Comment:
This is a very generic name. I suggest to make it very specific for contract input. I think we even have a `createPactCommand` function somewhere else already